Pod/Classes/VOKCoreDataManager.m

```diff
@@ -161,8 +145,7 @@ - (void)initPersistentStoreCoordinator
                               NSInferMappingModelAutomaticallyOption: @YES,
                               };
     
-    NSError *error;
-    _persistentStoreCoordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:[self managedObjectModel]];
+    _persistentStoreCoordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:self.managedObjectModel];
```

Comment:
This may be out of scope for this refactor, buuuut
since `managedObjectModel` and `persistentStoreCoordinator` are now privately read-only properties (instead of raw ivars) would it make sense to change `initPersistentStoreCoordinator` and `initManagedObjectModel` to lazy getters?
If not, I think you can safely use `self.persistentStoreCoordinator` instead of the ivar in the two `init...` methods.